### PR TITLE
adding full django2.0 compatibility.

### PR DESCRIPTION
**Pull request by gandhi23**
adding full django2.0 compatibility by adding django2.0 test to .travis.yml and tox.ini, so full django2.0 is implemented. this is a part of my Google code in task of @openWISP oraganisation. link of google code-in task: https://codein.withgoogle.com/dashboard/task-instances/4641906212470784/

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -46,4 +47,4 @@ commands =
 #   tox -e vagrant-mysql
 commands =
     vagrant up
-    vagrant ssh -c "cd /vagrant && DJANGO_SETTINGS_MODULE=test_project.mysql_settings python runtests.py"
+    vagrant ssh -c "cd /vagrant && DJANGO_SETTINGS_MODULE=test_project.mysql_settings python runtests.py"
```
Comment (auvipy) [inline tox.ini]: a new line needed.

File: tox.ini [hunk 2]
```diff
@@ -20,6 +20,7 @@ deps =
     19: Django >= 1.9, < 1.10
     110: Django>=1.10,<1.10.99
     111: Django>=1.11a1,<1.11.99
```
Comment (auvipy) [inline tox.ini]: 111 could be django 1.11 rather then 1.11a1

**Comment by auvipy** [comment]
@gregmuellegger could you please check?

**Comment by gandhi23** [comment]
k

**Comment by gandhi23** [comment]
i will do this in a new PR

**Comment by gandhi23** [comment]
i lost acess to the repo

**Pull request closed without merging**